.github/workflows/build.yml

```diff
@@ -16,13 +16,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v1.10.0
```

Comment:
I don't see v1.10.0 in the releases: https://github.com/actions/setup-java/releases